src/backend/metal/src/device.rs

```diff
@@ -104,7 +104,7 @@ fn get_final_function(
     library: &metal::LibraryRef,
     entry: &str,
     specialization: &pso::Specialization,
-    function_specialization: bool,
+    function_specialization: bool
```

Comment:
leftover change? trailing comma is good

Reply:
I *think* i got the squashing correct, have never done that before. Also it took me ages to realise i should be running `rustfmt +nightly *` so i may have misformatted things before.

Reply:
hmm, the trailing comma is still missing here?